### DISCUSSION TITLE
TinyEngine生成的vue代码运行环境

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#88] [General]
我试了tiny-engine-generate-preview和repl现在都可以显示tinyEngine生成的页面了，然后我发现repl用的是iframe显示页面，这样跟页面交互很麻烦。而且以后我要用tab多页签显示多个页面就更麻烦了。另外一个需求是我用tinyEngine修改表单生成页面后，前端不用重新打包只要刷新就可以读取到修改后的新页面，不知是否有较好的解决方案？

**Comment by qian-chen97** [comment]
+1

**Comment by stwflyfox** [comment]
vue3-sfc-loader能用？

**Comment by chilingling** [comment]
需求记录下来了，有进展会及时更新，感谢关注
